test/functional/sqlsrv/sqlsrv_statement_datetimes_as_strings.phpt

```diff
@@ -139,6 +139,11 @@ $query = 'SELECT CONVERT(date, SYSDATETIME()), SYSDATETIME(),
                  CONVERT(time, SYSDATETIME())';
 $stmt = sqlsrv_query($conn, $query);
 $values = sqlsrv_fetch_array($stmt, SQLSRV_FETCH_NUMERIC);
+sqlsrv_free_stmt($stmt);
+sqlsrv_close($conn);
+
+// Connect again with ColumnEncryption data
+$conn = AE\connect(array('ReturnDatesAsStrings' => true));

```

Comment:
Why disconnect and then reconnect?

Reply:
The first connect is the regular one without ColumnEncryption